test/literal.ls

```diff
@@ -119,6 +119,20 @@ compile-throws 'unterminated words' 5 '''
   <[
   '''
 
+# [LiveScript#739](https://github.com/gkz/LiveScript/issues/739)
+# Don't interpret WORDS:\n as a newline
+w = <[
+]>
+eq w.length, 0
+
+# And don't interpret WORDS:++ as concat
+w = <[++]>
+eq w.0, \++
+
+# And don't interpret WORDS:. as compose
+# (Note: the spacing and punctuation in this test is significant)
+eq '' + <[.]> \.
```

Comment:
What happens if you put whitespace around `.` in the word list

Reply:
If there's whitespace on either side of the `.`, the issue isn't triggered (before and after this patch). The lexer packs everything, whitespace included, between the brackets into the `WORDS` token, and only got confused when the value of that token exactly matched some expected values of other types of token.

Reply:
Consider putting this explanation as comment in test (explaining why the spacing is significant)?